test.txt

```diff
@@ -1 +0,0 @@
-test for keep file for user command reassembler
```

Comment:
I remove test.txt here, dose anyone know why should we keep this file under code base root?